experiments/README.md

```diff
@@ -0,0 +1,23 @@
+# Experiments
+
+- RQ-1 How can we scale training/validating of geospatial models?
+  - Train models in reasonable amount of time
+- RQ-2 How can we parameterize the configuration of the model?
+  - Parameterizing models allows for users to specify predefined hyperparameters which saves time
+- RQ-3 How can we efficiently store and retrieve trained models?
```

Comment:
Minor comment: I don't think we are planning to test anything around how we store the models, but just the retrieval times, so we can probably just focus on retrieval in this question.

Reply:
Need to determine the best way to build this experiment (e.g., random sample of models to retrieve, base it on model size and retrieval time, etc.). Since we are only planning to set up retrieval in one way, will measuring this add much value to our project?

Reply:
We can just retrieve N models and measure network I/O for the machines